### ISSUE TITLE
Import with multiple aliases has an alias removed

### ISSUE THREAD

**Issue by bdukes** [avh4/elm-format#379]
Using elm-format-0.18 0.7.0-exp, I have a file with the following imports:

```
import Html.Attributes.Aria as Aria exposing (ariaLabel, ariaLabelledby, role)
import Html.Attributes.Aria as Html
```

After running elm-format, they're combined into 

```
import Html.Attributes.Aria as Aria exposing (ariaLabel, ariaLabelledby, role)
```

(depending on the order, the alias may be `Aria` or `Html`)

Is this intentionally not supported, or a bug in the sorting?

Thanks!

**Comment by stil4m** [comment]
Just for the sake of recording: I've reported this a few weeks ago [here](https://github.com/avh4/elm).

**Comment by avh4** [comment]
Yes, this is intentional in 0.7.0-exp.  The current sorting algorithm merges imports of the same module, and imports can only have one `as` clause, so it currently chooses the first.

**Comment by alvivi** [comment]
Is this going to be keep in next version? we use a lot this feature in order to avoid conflicts, for example, when using `map` from the `Html` module (exposed by `Html` and `Html.Attributes`)?

```
import Html as H
import Html.Attributes as H
import Html  -- Enable the use of Html.map 
```

**Comment by aforemny** [comment]
Hi, I have another example that I think belongs here:

(a):
```
import Html exposing (Html, text)
import Html as Html_
```

gets turned into

(b)
```
import Html as Html_ exposing (Html, text)
```

Rationale: I import both `Html` and `Html.Attributes` as `Html`, except when I very rarely need a `Html_.form`.

Bad habits aside, I think elm-format should recognize that (a) and (b) are not equivalent. I was hoping that

```
import Html as Html exposing (Html, text)
import Html as Html_
```

would work as a work-around, but it does not.
